app/models/subscription.rb

```diff
@@ -28,14 +28,6 @@ class Subscription
   attr_accessor :first_name, :last_name, :zip, :language, :tdd
 
   validate :email_or_phone?
```

Comment:
Is CodeRed doing an email validation check? Do those errors look ok when they happen?

What about the case where someone checks either call or text but doesn't add a number?